ollama/engine.py

```diff
@@ -39,16 +38,14 @@ def load(model_name, models={}):
             for model_type in cls.model_types()
         }
 
-        while len(runners) > 0:
+        for match, _ in process.extract(model_path, runners.keys(), limit=len(runners)):
```

Comment:
second part of the tuple is ratio/score which doesn't matter a whole lot in this case